backend/app/agents/devrel/nodes/gather_context.py

```
@@ -28,8 +30,29 @@ async def gather_context_node(state: AgentState) -> Dict[str, Any]:
+        avatar_url = author.get("avatar") or author.get("avatar_url")
+
+        if discord_id:
+            try:
+                user = await get_or_create_user_by_discord(
+                    discord_id=str(discord_id),
+                    display_name=str(display_name or discord_username or discord_id),
+                    discord_username=str(discord_username or display_name or discord_id),
+                    avatar_url=avatar_url,
+                )
+                profile_data = user.model_dump()
+            except Exception as exc:  # pragma: no cover - graceful degradation
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)